spec/lib/miq_automation_engine/miq_ae_object_spec.rb

```diff
@@ -1,3 +1,5 @@
+include AutomationSpecHelper
```

Comment:
@Fryguy 
The AutomationSpecHelper is only being used in specs, its not being used under lib/miq_automation/engine